xclim/indices/_agro.py

```diff
@@ -507,6 +508,133 @@ def cool_night_index(
     return cni
 
 
+@declare_units(pr="[precipitation]", evspsblpot="[precipitation]")
+def dryness_index(
+    pr: xarray.DataArray,
+    evspsblpot: xarray.DataArray,
+    lat: xarray.DataArray | str | None = None,
+    wo: str = "200 mm",
+    freq: str = "YS",
+) -> xarray.DataArray:
+    """Dryness Index.
+
+    Mean minimum temperature for September (northern hemisphere) or March (Southern hemisphere).
+    Used in calculating the Géoviticulture Multicriteria Classification System.
+
+    Warnings
+    --------
+    Dryness Index expects CF-Convention conformant potential evapotranspiration (positive up).
+
+    Parameters
+    ----------
+    pr : xarray.DataArray
+        Precipitation.
+    evspsblpot : xarray.DataArray
+        Potential evapotranspiration.
+    lat : xarray.DataArray or {"north", "south"}, optional
+        Latitude coordinate as an array, float or string.
+        If None, a CF-conformant "latitude" field must be available within the passed DataArray.
```

Comment:
In any case, method `.replace` will fail if the passed DataArrays don't have cftimes, correct? I think this is unsual for the data we work with? Maybe it should be said somewhere in the docstring too? Or add a conversion step if needed?

Reply:
Woups sorry @Zeitsperre, I didn't know about that we I suggested it! 

There are hundred of ways to do this, I just came up with:
```
di_south['time'] = di_south.indexes['time'].shift(-6, 'MS')
```

Reply:
I'll _replace_ the `replace` line with Pascal's suggestion!